File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// PrankShow SDK exists test

require_once __DIR__ . '/../prankshow_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = PrankShowSDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}

File: rb/test/exists_test.rb
# PrankShow SDK exists test

require "minitest/autorun"
require_relative "../PrankShow_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = PrankShowSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
